### DISCUSSION TITLE
Option to assign a "sorting title" to a game entry

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#1062] [Ideas]
This would allow us to do the following:
- Ignore definite and indefinite articles at the start of titles so we don't have to rename them to force the correct sorting order.
- Actually have titles that use roman numerals display in the correct order instead of having 9 come before 5. `I, II, III, IV, IX, V`!?

![321](https://user-images.githubusercontent.com/45054151/180736916-41f2a6ac-0444-4d0b-81bb-5877866662cd.PNG)

**Comment by Blazekickn** [comment]
Itd be nice to have an option to show the sorting title instead of the display one

**Comment by Xeinok-Pals** [comment]
Would be nice to have and I wonder if this might be a roundabout fix for the awkward searching on RA where searching for "The... X" will fail and you need to currently search in way such as: "Last Blade 2, The"

**Comment by hb-robo** [comment]
Asking mostly out of genuine curiosity: is a manually assigned sorting title necessary for this, or can the search function be improved to the point where the problems regarding token order, Roman numerals, and "The" are automatically resolved?

It seems like there might be a way that this could be done at runtime without requiring devs to tag these sorting titles, but I admit to not being really informed about search engines in general.

**Comment by wescopeland** [comment]
https://github.com/RetroAchievements/RAWeb/pull/2690
